### ISSUE TITLE
Change away from Cloud9 to Codeanywhere for Chromebook development?

### ISSUE THREAD

**Issue by charlesdwesley** [DjangoGirls/tutorial#1081]
Hi there! We've had issues with people on Chromebooks not wanting to sign up for Cloud9, due to the requirement that you enter your credit card information. I spent some time looking at alternatives, and it looks like Codeanywhere provides all of the same features as Cloud9, without the credit card info requirement.

I might have time during our upcoming DjangoGirls event to make a pull request updating the English Chromebook tutorial to use Codeanywhere instead, but I wanted to check first, so I don't spend a bunch of time on an update that we wouldn't use anyway. What do you think about making this tutorial switch?

**Comment by aniav** [comment]
Sounds awesome! Go on :)

And sorry for not responding earlier!

**Comment by vilmibm** [comment]
This has been on my todo list for ages D: I'm working on those changes right now

**Comment by vilmibm** [comment]
(history: initially cloud9 didn't require a CC which is why I chose it for the tutorial. really bummed about them changing like that.)

**Comment by thibaudcolas** [comment]
I have received an email from CodeAnyhwere which says they now limit the free user tier to 2h/day of container usage (not the IDE, but the running server). It's not super clear to me how the usage is calculated (e.g. if the container turns on whenever you're logged in, or if it turns off after some inactivity), but either way it doesn't sound like enough for a workshop.

**Comment by das-g** [comment]
> I wonder if it would be viable to use PythonAnywhere instead of Cloud9. I don't think it's as good IDE-wise, but it has the big advantage of already being relied on for the workshop.

@hjwp What's your opinion on that?

**Comment by kewtree1408** [comment]
Hey everyone,
I have an upcoming DjangoGirls event in Amsterdam, and I’ve noticed that AWS has deprecated Cloud9:

> "After careful consideration, we have made the decision to close new customer access to AWS Cloud9, effective July 25, 2024."

Here is the [source](https://aws.amazon.com/blogs/devops/how-to-migrate-from-aws-cloud9-to-aws-ide-toolkits-or-aws-cloudshell/ ) link.

[GitHub Codespace](https://github.com/features/codespaces) was tested by me and one of my attendees. She confirmed that it works and that it seems easy to set up, as a GitHub account needs to be created anyway.

I'm preparing updated documentation that will use. 

@das-g Let me know your opinion regarding this.

**Comment by das-g** [comment]
At the moment we seem to have [instructions for 3 cloud IDE platforms](https://github.com/DjangoGirls/tutorial/blob/4ab5fdb55203a86fb84833867dce1689709e61cf/en/chromebook_setup/instructions.md):
- PaizaCloud Cloud IDE
- AWS Cloud9
- Glitch.com

If Cloud9 isn't available anymore, we definitely need to remove that.

If PaizaCloud and Glitch.com continue to be viable options (Has anyone tried them recently for the Django Girls tutorial?), replacing Cloud9 with a new platform isn't absolutely necessary. Though, if GitHub Codespaces is well-suited for the tutorial, adding it might be a good idea, anyway. (And not requiring yet another account is definitely a plus!)

**Comment by kewtree1408** [comment]
I've tried both `PaizaCloud Cloud IDE` and `Glitch.com`. They are Cloud IDEs, but I didn't fully understand them when I was reading the tutorial. I thought I have to setup all 3: `PaizaCloud Cloud IDE` and `Glitch.com` and `Cloud 9`.

- I've created `PaizaCloud IDE` yesterday and today I see the following message:
<img width="1435" alt="Screenshot 2024-09-19 at 17 03 26" src="https://github.com/user-attachments/assets/bbc29a02-2404-4eb8-b288-c572b2a00ab1">
I suppose they changed something as well.


- I've started following `Glitch.com` and I personally found it hard to follow. Some of the instructions mixed up with settings and files, that are not created yet.

The advantages of the GitHub Codespace, that I see, are:
- you don't have to create extra account
- you don't have to setup git profile settings
- bash terminal is ready to use, so the student can follow the rest of the instructions for linux/ubuntu
